### PR TITLE
Crew license framework

### PR DESCRIPTION
**Pull request by saltedcoffii**
Chromebrew doesn't have a license field in packages, but its important to some users. This adds the framework to add the `license ''` field to chromebrew packages. When this gets accepted, I'll start going around to all the `package.rb` files and adding this field.

### CODE REVIEW

**Comment by uberhacker** [comment]
Don't you also need to add a package property in https://github.com/skycocker/chromebrew/blob/master/lib/package.rb?  If we do this, then we need to either have a default value set in each package or a way to ignore the property if a value is not set.

**Comment by saltedcoffii** [comment]
I don't know what else needs to happen. I could run a `sed` on the packages directory to add the line `license 'undefined'`. Would that work?

**Comment by uberhacker** [comment]
If you did that, you might as well put in the correct license.  I think undefined would confuse users.

**Comment by uberhacker** [comment]
I resolved a merge conflict in openssh.

File: packages/desktop_file_utils.rb [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+  license 'GPL-2+'
+  compatibility 'all'
+  source_url 'https://www.freedesktop.org/software/desktop-file-utils/releases/desktop-file-utils-0.23.tar.xz'
+  source_sha256 '6c094031bdec46c9f621708f919084e1cb5294e2c5b1e4c883b3e70cb8903385'
+
+  binary_url ({
+    aarch64: 'https://dl.bintray.com/chromebrew/chromebrew/desktop_file_utilities-0.23-chromeos-armv7l.tar.xz',
+     armv7l: 'https://dl.bintray.com/chromebrew/chromebrew/desktop_file_utilities-0.23-chromeos-armv7l.tar.xz',
+       i686: 'https://dl.bintray.com/chromebrew/chromebrew/desktop_file_utilities-0.23-chromeos-i686.tar.xz',
+     x86_64: 'https://dl.bintray.com/chromebrew/chromebrew/desktop_file_utilities-0.23-chromeos-x86_64.tar.xz',
```
Comment (uberhacker) [inline packages/desktop_file_utils.rb:16]: These binaries don't match the package name.  I see what you are trying to do but is it worth it just to shorten the name?  I imagine most of the time you would never explicitly install this package anyway.

Comment (saltedcoffii) [reply packages/desktop_file_utils.rb:16]: I did have to explicitly install it once, I forget why, and that's how I noticed that the name changed. I'll change it back in this PR but it shud be changed later

**Comment by saltedcoffii** [comment]
Omg looks like my master branch somehow got mixed into this branch and i pushed it without realizing. Lemme fix that 🤦🏻

File: packages/bcd.rb [hunk 2]
```diff
@@ -0,0 +1,99 @@
+require 'package'
+
+class Bcd < Package
```
Comment (uberhacker) [inline packages/bcd.rb:3]: New package?  Can we put this in a separate PR?

Comment (saltedcoffii) [reply packages/bcd.rb:3]: That's leftover, I forgot to get it in c9bf791, I'll take care of it in the next commit.

File: packages/fly.rb [hunk 3]
```diff
@@ -2,8 +2,9 @@
 
 class Fly < Package
   description 'fly is a C program that creates PNG, JPEG or GIF images on the fly from CGI and other programs.'
-  homepage 'http://martin.gleeson.com/fly/index.html'
+  homepage 'http://www.unimelb.edu.au/fly/fly.html' # 403 Forbidden
```
Comment (uberhacker) [inline packages/fly.rb:5]: I get a 404 on this page.  Change to http://www.w3perl.com/fly/?

File: packages/freebasic.rb [hunk 4]
```diff
@@ -1,16 +1,17 @@
...
-  description 'FreeBASIC is a free/open source (GPL), BASIC compiler for Microsoft Windows, DOS and Linux.'
+  description 'FreeBASIC is a free and open source, BASIC compiler for Microsoft Windows, DOS and Linux.'
   homepage 'https://www.freebasic.net/'
   version '1.07.1-1'
+  license 'LGPL-2 and GPL-2'
   compatibility 'i686,x86_64'
   case ARCH
   when 'i686'
-    source_url 'https://downloads.sourceforge.net/project/fbc/Binaries%20-%20Linux/FreeBASIC-1.07.1-linux-x86.tar.gz'
+    source_url 'https://downloads.sourceforge.net/project/fbc/Binaries%20-%20Linux/FreeBASIC-1.07.1-linux-x86.tar.gz' # 404 not found
```
Comment (uberhacker) [inline packages/freebasic.rb:11]: New version at https://sourceforge.net/projects/fbc/files/Binaries%20-%20Linux/FreeBASIC-1.07.2-linux-x86.tar.gz?

File: packages/freebasic.rb [hunk 5]
```diff
@@ -1,16 +1,17 @@
...
     source_sha256 'a5b8f31e38a214d63a3d8e0b1ac6608f6a304891e40a3c0755940cba2f8401f1'
   when 'x86_64'
-    source_url 'https://downloads.sourceforge.net/project/fbc/Binaries%20-%20Linux/FreeBASIC-1.07.1-linux-x86_64.tar.gz'
+    source_url 'https://downloads.sourceforge.net/project/fbc/Binaries%20-%20Linux/FreeBASIC-1.07.1-linux-x86_64.tar.gz' # 404 not found
```
Comment (uberhacker) [inline packages/freebasic.rb:14]: New version at https://sourceforge.net/projects/fbc/files/Binaries%20-%20Linux/FreeBASIC-1.07.2-linux-x86_64.tar.gz?

Also, arm version available now?
https://sourceforge.net/projects/fbc/files/Binaries%20-%20Linux/FreeBASIC-1.07.2-linux-arm.tar.gz

File: packages/freebasic.rb (hunk 4 above)
Comment (saltedcoffii) [reply packages/freebasic.rb:11]: Do you think we shud get this pr merged first and then fix that? New versions requires testing

Comment (uberhacker) [reply packages/freebasic.rb:11]: Yes, makes sense to include in a separate PR.

**Comment by saltedcoffii** [comment]
Oh my gosh it's done 😭😭😭
